sidebars.js

```diff
@@ -695,14 +704,6 @@ module.exports = {
         id: "components/early-access/overview",
       },
       items: [
-        {
-          type: "category",
-          label: "Experimental features",
-          items: [
-            "components/early-access/experimental/rpa/rpa-integration",
```

Comment:
@marstamm Do you want to keep these docs, at all? Do they still serve a purpose for the users, or should these pages also be removed entirely?

Reply:
These pages should be removed from 8.7 onwards, once we have the getting started guide in place

Reply:
Done 👍